addons/external_intercom/CustomInteractionPoints.hpp

```diff
@@ -0,0 +1,97 @@
+// Vanilla NATO
+class B_APC_Tracked_01_base_F;
+class B_APC_Tracked_01_AA_F : B_APC_Tracked_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.56836,-4.79979,-1.1637};
+};
+
+class AFV_Wheeled_01_up_base_F;
+class B_AFV_Wheeled_01_up_cannon_F : AFV_Wheeled_01_up_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.21875,-4.34796,-0.82896};
+};
+
+class AFV_Wheeled_01_base_F;
+class B_AFV_Wheeled_01_cannon_F : AFV_Wheeled_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.21875,-4.34796,-0.82896};
+};
+
+class B_APC_Tracked_01_base_F;
+class B_APC_Tracked_01_CRV_F : B_APC_Tracked_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.55518,-4.79966,-0.811272};
+};
+
+class B_APC_Tracked_01_base_F;
+class B_APC_Tracked_01_rcws_F : B_APC_Tracked_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.54883,-4.79953,-0.84446};
+};
+
+class B_APC_Wheeled_01_base_F;
+class B_APC_Wheeled_01_cannon_F : B_APC_Wheeled_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {0.566895,-4.39741,-0.893777};
+};
+
+class B_MBT_01_mlrs_base_F;
+class B_MBT_01_mlrs_F : B_MBT_01_mlrs_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.31592,-4.8209,-1.43332};
+};
+
+class B_MBT_01_arty_base_F;
+class B_MBT_01_arty_F : B_MBT_01_arty_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.31592,-4.8209,-1.43332};
+};
+
+class B_MBT_01_base_F;
+class B_MBT_01_cannon_F : B_MBT_01_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {1.28955,-4.39827,-1.06652};
+};
+
+// Vanilla CSAT
+class O_APC_Tracked_02_base_F;
+class O_APC_Tracked_02_AA_F : O_APC_Tracked_02_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {0.936035,-4.94368,-0.984507};
+};
+
+class APC_Wheeled_02_base_v2_F;
+class O_APC_Wheeled_02_rcws_v2_F : APC_Wheeled_02_base_v2_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {0.670898,-4.32084,-0.725672};
+};
+
+class O_APC_Tracked_02_base_F;
+class O_APC_Tracked_02_cannon_F : O_APC_Tracked_02_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {0.964355,-4.93802,-0.765622};
+};
+
+class O_MBT_02_arty_base_F;
+class O_MBT_02_arty_F : O_MBT_02_arty_base_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {-1.24561,-5.41896,-1.4492};
+};
+
+class Tank_F;
+class MBT_04_base_F : Tank_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {-1.44043,-5.65282,-1.23575};
+};
+
+class Tank_F;
+class MBT_02_base_F : Tank_F {
+    TFAR_ExternalIntercomInteractionPoint[] = {-1.26074,-4.75584,-0.938046};
```

Comment:
would be cool if this can also support string memory point

Reply:
Good point!